### PR TITLE
chore(statics) : onboard new tokens

### PR DESCRIPTION
**Pull request by hitansh-madan**
Ticket: EA-1314

### CODE REVIEW

**Review by OttoAllmendinger — COMMENTED** [review]
flush

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -5034,6 +5050,14 @@ export const coins = CoinMap.fromCoins([
     '0xa31b1767e09f842ecfd4bc471fe44f830e3891aa',
     UnderlyingAsset.ROOBEE
   ),
+  erc20(
+    'a72f89b9-7c43-4547-8d4e-9f6d10867347',
+    'rpl',
+    'Rocket Pool Protocol',
```
Comment (aarvay) [inline modules/statics/src/coins.ts:5056]:
```suggestion
    'Rocket Pool',
```

**Review by aarvay — CHANGES_REQUESTED** [review]

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (hitansh-bitgo) [reply modules/statics/src/coins.ts:5056]: Does this not need to match the value for name() function from the contract?

Comment (aarvay) [reply modules/statics/src/coins.ts:5056]: Not a hard rule.

**Review by aarvay — APPROVED** [review]

**Pull request merged**